src/less/components/action-list.less

```diff
@@ -60,6 +60,7 @@
         &:focus {
             cursor: pointer;
             background-color: @light-gray;
+            border: 2px solid transparent;
```

Comment:
Something is funky in the focus state for the action-list icon. Same for the close button I earlier mentioned.

Reply:
Never mind this